.nvmrc

```diff
@@ -0,0 +1 @@
+lts/*
```

Comment:
build was failing for netlify. need to notify to use latest version using this .nvmrc
https://docs.netlify.com/configure-builds/manage-dependencies/